P5/Source/Guidelines/en/CO-CoreElements.xml

```diff
@@ -85,26 +85,39 @@ organization of the module described here. This should be read in
-cannot appear except within one, from <term>chunks</term>, which can
-appear between, but not within, paragraphs, and from
-<term>inter-level</term> elements, which can appear either within a
-single paragraph or between paragraphs. The class of phrases includes
-emphasized or quoted phrases, names, dates, etc. The class of
-inter-level elements includes bibliographic citations, notes, lists,
-etc. The class of chunks includes the paragraph itself, and other
-elements which have similar structural properties, notably the
-<gi>ab</gi> (anonymous block) element described in <ptr target="#SASE"/>) which may be used as an alternative to the paragraph
-in some kinds of texts.</p>
+  <p>Paragraphs in modern printed or online text are typically visually 
+    offset with whitespace or an indented first line. But paragraphs are 
+    not simply blocks of text. The paragraph may be thought of as a mid-level 
+    unit of sense, a coherent grouping of sentences. Paragraphs may, in turn, 
+    be grouped into larger divisions, such as chapters. Because it is a unit 
+    of sense rather than simply a block of text, the <gi>p</gi> element in TEI 
+    may contain other structures displayed as blocks, such as lists or quotes. 
+    This distinguishes it from the <ref 
+      target="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p">p</ref> 
+    element in HTML, which is primarily a block of text, and from the <gi>ab</gi> 
+    (anonymous block) element described in <ptr target="#SASE"/>) which may be used 
+    as an alternative to the paragraph in cases that require a chunk-level container 
+    which is not necessarily a sense unit and which may have different structural properties.</p>
+  <p>The paragraph can be understood in the context of the distinct forms of textual 
+    division discussed in <ptr target="#STBTC"/>: <term>chunk</term>, 
+    <term>phrase</term>, and <term>inter</term>-level. 
+    Chunk-level elements are paragraphs and other elements which have similar 
+    structural properties. Phrase-level elements must be entirely contained within a 
+    paragraph or other chunk-level structure. This type includes emphasized or quoted 
+    phrases, names, dates, etc. Inter-level elements can appear either within a 
+    paragraph or between paragraphs, and include bibliographic citations, notes, and 
+    lists. The <gi>ab</gi> (anonymous block) element is an alternative to 
+    <gi>p</gi> which is useful in cases where paragraphs are not present, but chunk-level 
+    organization is still needed. <gi>ab</gi> may contain other <gi>ab</gi>s, 
+    may use the <att>type</att> attribute, and does not necessarily represent a coherent set of 
+    statements.</p>
+  <p>Paragraphs can contain any of the other elements described within this chapter, 
+    as well as some other elements which are specific to individual text types. Because 
+    paragraphs may appear in different base or additional tag sets, their possible 
```

Comment:
The concepts of “base tag set” and “additional tag set” were P4 ideas that, it seems to me, are not that relevant any more.